docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -168,8 +168,24 @@ include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 
 <1> `HotBackup` resource name used for restore. The backup folder name will be taken from the `HotBackup` resource.
 <2> Agent responsible for restoring data from the local storage.
-    The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
-    uses the latest agent version that is compatible with its version.
+The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
+uses the latest agent version that is compatible with its version.
+
+NOTE: For restore to be successful you have to give restore configuration at the `Hazelcast` custom resource creation. Otherwise, triggering restore on currently running members will likely to fail.
+
+To restore from local backups that is not taken by HotBackup resource, configure `localConfig`.
+
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hazelcast-persistence-restore-local.yaml[]
+----
+
+<1> For restore to be successful, name of the cluster that the backup taken and restored must be the same.
+<2> `baseDir` is the root directory of persistence.
+<3> `backupDir` is the directory for backups.
+<4> `backupFolder` is the folder for a specific backup.
+
+NOTE: Agent copies backup to be restored from `baseDir` + `backupDir` + `backupFolder` to `/data/hot-restart` path.
```

Comment:
```suggestion
NOTE: The agent copies the backup to be restored from `baseDir`, `backupDir`, and `backupFolder` to the `/data/hot-restart` path.
```

Reply:
my intention while using `+` here is to explain that the agent joins those parameters, `,` does not give that idea IMO. Do you have any other idea?

Reply:
https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/171/commits/42d63b960040c5395c3c8e7bf8d026dca387231a